modules/ROOT/pages/lb-create-cli.adoc

```diff
@@ -20,23 +20,48 @@ Before you create a load balancer:
 * Create an Anypoint Virtual Private Cloud (Anypoint VPC) instance in the organization where you want to create a DLB.
 * Create at least one certificate and private key for your certificate.
 
-== Create a Load Balancer
+== Create a Load Balancer Using the CLI
 
 Use this command to create a DLB from the CLI:
 
+[source,linenums]
 ----
-cloudhub load-balancer create myVPC myLB_name example-com-crt.pem example-com-private.pem
+cloudhub load-balancer create myVPC myLB_name certificateName privateKey
 ----
 
-This command specifies the Anypoint VPC instance in which to create the DLB and the certificate and private keys that the DLB uses.
+This command specifies the Anypoint VPC instance _myVPC_ in which to create the DLB and the _certificateName_ and _privateKey_ that the DLB uses.
 
 //NO RENAME SHARED
 include::partial$no-rename.adoc[tag=noRenameDLB]
 
+== Enable TLS 1.0 on a DLB Using the CLI
+
+// TLS 1.0 Support
+include::partial$tls-support.adoc[tag=tlsSupport]
+
+To enable TLS 1.0 support on a DLB using the CLI: 
+
+. Run the following command to recreate the SSL endpoint for the _myLB_name_ load balancer and add TLS 1.0 support: 
++
+[source,linenums]
+----
+cloudhub load-balancer ssl-endpoint add --tlsv1 myLB_name certificateName privateKey
+----
+. Run the following command to set the _certificateName_ as the default SSL endpoint for the _myLB_name_ load balancer: 
```

Comment:
```suggestion
. Run the following command to set _certificateName_ as the default SSL endpoint for the _myLB_name_ load balancer: 
```
why ital instead of mono? GLOBAL